book/templating.rst

```diff
@@ -1012,13 +1018,18 @@ but Symfony provides a more dynamic option via the ``asset`` Twig function:
 
         <img src="{{ asset('images/logo.png') }}" alt="Symfony!" />
 
-        <link href="{{ asset('css/blog.css') }}" rel="stylesheet" type="text/css" />
+        <link href="{{ asset('css/blog.css') }}"
+            rel="stylesheet"
+            type="text/css" />
```

Comment:
I was thinking the same thing, but unfortunately the PHP example without type would still (I think) have the scrollbar.